lms/data_tasks/report/create_from_scratch/05_entities_decorated/01_users/01_create_view.jinja2.sql

```diff
@@ -2,6 +2,45 @@ DROP MATERIALIZED VIEW IF EXISTS report.users CASCADE;
 
 CREATE MATERIALIZED VIEW report.users AS (
     WITH
+        user_modify_dates AS (
+            -- Combine all the relevant users together to create synthetic
+            -- created and updated dates.
+            SELECT
+                user_map.user_id,
+                MIN(lms_user.created) AS created_date,
+                MAX(lms_user.updated) AS updated_date
+            FROM report.user_map
+            JOIN "user" AS lms_user ON
+                lms_user.id = user_map.lms_user_id
+            GROUP BY user_map.user_id
+        ),
+
+        user_activity_dates AS (
+            SELECT
+                user_id,
+                MAX(last_activity_date) AS last_activity_date
```

Comment:
What about basing it completely on activity as defined here:

- remove updated_date
- remove created_date (which I guess HS also tracks its own record's metadata)
- add `first_activity_date` as the min() here.